tests/data_layer/test_data_interface.py

```diff
@@ -0,0 +1,169 @@
+"""Test data interface
+"""
+from datetime import datetime
+
+from pytest import fixture
+from smif.data_layer import DatafileInterface
+
+
+@fixture(scope='function')
+def get_sos_model_run():
+    """Return sample sos_model_run
+    """
+    return {
+        'name': 'unique_model_run_name',
+        'description': 'a description of what the model run contains',
+        'stamp': datetime(2017, 9, 20, 12, 53, 23),
```

Comment:
I'm not sure this will get written correctly?  Prefer ISO8601 string written to yaml files, and then parsed to datetime object in the python dict.

Reply:
I did a quick check. And this is correct parsed back into a datatime object by the yml parser.

---------------------------- Captured stdout call -----------------------------
```python
{'name': 'sos_model_run2', 'decision_module': 'energy_moea.py', 'narratives': [{'technology': ['Energy Demand - High Tech']}, {'governance': 'Central Planning'}], 'stamp': datetime.datetime(2017, 9, 20, 12, 53, 23), 'description': 'a description of what the model run contains', 'scenarios': [{'population': 'High Population (ONS)'}], 'timesteps': [2015, 2020, 2025], 'sos_model': 'energy'}
```

Reply:
In this files it will look like
```yaml
stamp: 2017-09-20 12:53:23
```
So yes, converting it to an ISO8601 string is definitely better. I will change it 👍

Reply:
But yes, the ISO8601 was specified and contain more information. I will change it.